src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/instance_blackrock_spire.cpp

```diff
@@ -28,6 +28,8 @@
 
 uint32 const DragonspireMobs[3] = { NPC_BLACKHAND_DREADWEAVER, NPC_BLACKHAND_SUMMONER, NPC_BLACKHAND_VETERAN };
 
+const Position finkleSpawnPosition = { 58.259682f, -535.346008f, 110.935753f, 3.247705f };
+
```

Comment:
Unused after e6878a60474bbb34f6bf2e7fd81aaf259c6ca341. Also `bool finkleSpawned` is unused too.